### ISSUE TITLE
Tabbed Interface - Carousel: number of items not spoken using jaws

### ISSUE THREAD

**Issue by dfarough** [wet-boew/wet-boew#6948]
when tabbing through controls of the tabbed interface and Carousels, Jaws does not speak the number of items.  I expect to hear item 1 of 3 or some such information.  Instead Jaws just speaks item.  If I want to hear the rest, I have to press down arrow to hear the rest of the information.  When using NVDA I hear all of the information as expected. This behavior occurs  using Jaws 14, 15, and Jaws 16 (Current version.  This occurs using Internet explorer 10, and Firefox version 36.0.1.

**Comment by dfarough** [comment]
@wet-boew/access-working-group 
Sorry about forgetting to include the access working group tag.

**Comment by pjackson28** [comment]
@dfarough We are implementing it according to the WAI-ARIA tabpanel design pattern (http://www.w3.org/TR/wai-aria-practices/#tabpanel). It seems that NVDA is handling this correctly while Jaws is not. Do you have any examples to provide that implement the WAI-ARIA tabpanel design pattern and work as expected in Jaws? Or perhaps is this one of the several WAI-ARIA related issues that should be fixed in Jaws instead?

**Comment by hsrudnicki** [comment]
@pjackson28 @dfarough @LaurentGoderre : Please update the status of this issue whether this issue can be closed or otherwise?

**Comment by hsrudnicki** [comment]
@dfarough and others, please update this issue's status, thx

**Comment by hsrudnicki** [comment]
No further discussion to this issue, lingering for 5months, so this issue labeled "Going Stale" and closed as resolved based on @pjackson28 response, that the issue is caused by one of 3rd parties, or if otherwise? 
Either way, please update this issue's status, thank you

**Comment by mysticjeff** [comment]
Should these types of issues be closed until resolved?

-----Original Message-----

**Comment by hsrudnicki** [comment]
Finally commented, thank you, re-opened but how long?
Please keep this issue's status updated.

**Comment by mysticjeff** [comment]
Someone would need to commit to fixing it before we could say how long.

-----Original Message-----

**Comment by hsrudnicki** [comment]
@mysticjeff : Thank you for your comment(s), and have you read last comment from @pjackson28 on which this issue had been closed then reopened due to your last comments, and who will be the significant in solving this issue ? Anyway, please keep this issue updated

**Comment by dfarough** [comment]
I just checked this component using Jaws 17 (latest version).  Jaws is now speaking the necessary info I.e. item 1 of 3).  I noticed when running Jaws 16 that there was an update available so I installed it.  Unfortunately this issue still is present even after installing the update for version 16.  Therefore, only users of the most recent version of jaws will see this information automatically when this control gains focus.

**Comment by hsrudnicki** [comment]
So, based on the last comment, can be this closed or otherwise? Either way, please update the status of this for the current year as well, thank you

**Comment by masterbee** [comment]
@dfarough - due you mind giving this another test? It is several months later and there has probably been more advancements and patches from screen readers to adopt the standards.

@mysticjeff - I am all for trying to help with assistive technologies and we do this a lot in WET. Yet it becomes a bit trickier (not to mention more technically expensive) when we try to accommodate screen-readers and other technologies (browsers as well) that are working towards standards. 

What are your thoughts on creating a bit of grid to target for? WET has historically had a family of browsers and there versions that we strive to develop for (more recent and a few versions back), and it appears that screen readers are following the same type of pattern in terms of compliance.
